sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1176,6 +1176,12 @@ def _add_argparse_args(cls, parser):
         default=0,
         help='Port to use for artifact staging. 0 to use a '
         'dynamic port.')
+    parser.add_argument(
+        '--java_launcher',
+        default='java',
+        help='The Java Application Launcher executable file to use for '
+        'starting a Java job server. If unset, `java` from the '
+        'environment\'s $PATH is used.')
```

Comment:
you noted the expansion service isn't affected by this PR - _should_ this arg affect the expansion service launcher as well, or should that be configured separately?

If the former why not go ahead and wire it up
If the latter this should be `java_job_server_launcher` (or something) to disambiguate from a possible future param for expansion services.

Reply:
That's a great question. Originally I was going to do both in this PR, but I reconsidered because expansion services and job servers are logically separate things (although job servers sometimes include expansion services). So I'll take your suggestion to rename, and we'll track the expansion service in BEAM-11414.